legend-engine-xts-java/legend-engine-xt-javaGeneration-pure/src/main/resources/core_external_language_java/metamodel_factories.pure

```diff
@@ -2874,6 +2874,40 @@ function meta::external::language::java::factory::knownPackage(pkg: meta::extern
    )))))))));
 }
 
+function meta::external::language::java::factory::isKnownPackage(pkg: meta::external::language::java::metamodel::Package[1]):Boolean[1]
```

Comment:
Can probably clean this up with Pierre's new-ish multi-if support:

https://github.com/finos/legend-pure/blob/master/legend-pure-functions/legend-pure-functions-base/legend-pure-m2-functions-base-pure/src/main/resources/platform_functions/lang/if.pure#L15